### ISSUE TITLE
CASTEP in parallel not running for more than 6 hours

### ISSUE THREAD

**Issue by mt1cmh** [rcgsheffield/sheffield_hpc#297]
Hi,

I've been playing with running larger calculations on Iceberg, but even when I have requested longer run times than 6 hours, my jobs crash out. I suspect I am using the wrong queue for it. I had though the mpi queue was correct.

Or, is this related to a more recent issue as CASTEP seems to be crashing out when it has been writing a check point file?

Hopefully the more I can get larger jobs to run, the better the online documentation can become for PhD students etc.

Thanks,

Chris

**Comment by willfurnass** [comment]
@mt1cmh Presumably this issue is no longer relevant and can be closed?
